File: Trivia/SignupRequest.cpp
#include "SignupRequest.h"

SignupRequest::SignupRequest(std::string username, std::string password, std::string email) : _username(username), _password(password), _email(email)
{}

std::string SignupRequest::getUsername()
{
	return _username;
}

std::string SignupRequest::getPassword()
{
	return _password;
}

std::string SignupRequest::getEmail()
{
	return _email;
}

File: TriviaClient/Requests/SignupRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace TriviaClient.Requests
{
    class SignupRequest
    {
        [JsonProperty(PropertyName ="username")]
        private string username;
        [JsonProperty(PropertyName = "password")]
        private string password;
        [JsonProperty(PropertyName = "email")]
        private string email;

        public SignupRequest(string username, string password, string email)
        {
            this.username = username;
            this.password = password;
            this.email = email;
        }

        public string GetUsername()
        {
            return username;
        }

        public string GetPassword()
        {
            return password;
        }

        public string GetEmail()
        {
            return email;
        }
    }
}
